tests/core/Observer.spec.ts

```diff
@@ -141,15 +143,16 @@ describe('Observer', async () => {
 
   it('should process a valid operation batch successfully.', async () => {
     // Prepare the mock response from the DownloadManager.
-    const didDocumentTemplate = require('../json/didDocumentTemplate.json');
-
     const [publicKey1, privateKey1] = await Cryptography.generateKeyPairHex('#key1', KeyUsage.recovery);
-    const [publicKey2, privateKey2] = await Cryptography.generateKeyPairHex('#key2', KeyUsage.recovery);
-    const operations = [
-      await OperationGenerator.generateCreateOperation(didDocumentTemplate, publicKey1, privateKey1),
-      await OperationGenerator.generateCreateOperation(didDocumentTemplate, publicKey2, privateKey2)
+    const signingKeys = await Cryptography.generateKeyPairHex('#key2', KeyUsage.signing);
+    const publicKeys = [
+      publicKey1,
+      signingKeys[0]
     ];
-    const operationsBuffer = operations.map((op) => { return Buffer.from(JSON.stringify(op)); });
+    const service = OperationGenerator.createIdentityHubServiceEndpoints(['did:sidetree:value0']);
+    const document = Document.create(publicKeys, service);
+    const operationsBuffer = [await OperationGenerator.createOperationBuffer(OperationType.Create, document, '#key1', privateKey1)];
+
```

Comment:
As per our offline conversation, maybe you can create the following function in OperationGenerator.ts which will be more explicit than the previous one and still be reusable. 

````
async createOperationBuffer(
  recoveryPublicKey: DidPublicKeyModel, 
  recoveryPrivateKey: string, 
  signingPublicKey: DidPublicKeyModel, 
  signingPrivateKey: string, 
  serviceEndpoints: string[]) : Promise<Buffer>{
...
}